api/rest/workshop_lectors.py

```diff
@@ -1,11 +1,14 @@
 from rest_framework import serializers, viewsets
 
-from .lectors import LectorSerializer
 from ..models import WorkshopLector
 
 
 class WorkshopLectorSerializer(serializers.ModelSerializer):
-    lector = LectorSerializer()
```

Comment:
@just-paja You don't get lector pohotos, about and other parameters in `/api/workshops/`, I hope you know that.